apps/docs/docusaurus.config.js

```diff
@@ -52,6 +52,17 @@ module.exports = {
     ],
   ],
   tagline: 'Performance & DX',
+  themes: [
+    [
+      require.resolve("@easyops-cn/docusaurus-search-local"),
+      {
+        hashed: true,
+        indexDocs: true,
+        indexPages: true,
+        language: ['en' ],
```

Comment:
nit: Leave out the extraneous space.
```suggestion
        language: ['en'],
```